src/emails/templates/EmailFooter.tsx

```diff
@@ -137,107 +137,115 @@ export const RedesignedBreachEmailFooter = (props: Props) => {
-            width="36px"
-            height="36px"
-            align="center"
-          />
-          <mj-text
-            color="#321C64"
-            font-size="22px"
-            font-weight="700"
-            align="center"
-          >
-            {l10n.getString("email-footer-support-heading")}
-          </mj-text>
-          <mj-text font-size="16px" font-weight="400" align="center">
-            {l10n.getFragment("email-footer-support-content", {
-              elems: {
-                "support-link": (
-                  <a
-                    href={supportLinkUrlObject.href}
-                    style={{ color: "#0060DF" }}
-                  />
-                ),
-              },
-            })}
-          </mj-text>
-        </mj-column>
-      </mj-section>
-      <mj-section padding-top="32px">
-        <mj-column>
-          <mj-image
-            alt={l10n.getString("email-footer-logo-mozilla-alt")}
-            src={`${process.env.SERVER_URL}/images/email/mozilla-logo-bw.png`}
-            href={`https://www.mozilla.org/?utm_source=monitor-product&utm_medium=email&utm_campaign=${props.utm_campaign}&utm_content=header-logo`}
-            width="150px"
-            align="center"
-          />
-          <mj-text
-            color="#0C0C0D"
-            font-size="14px"
-            line-height="21px"
-            font-weight="400"
-            align="center"
-          >
-            149 New Montgomery St, 4th Floor
-            <br />
-            San Francisco, CA 94105
-          </mj-text>
-          <mj-text
-            color="#0C0C0D"
-            font-size="14px"
-            line-height="21px"
-            font-weight="400"
-            align="center"
-          >
-            {l10n.getString("email-footer-trigger-transactional")}
-          </mj-text>
-          <mj-text
-            font-size="14px"
-            line-height="21px"
-            font-weight="400"
-            align="center"
-          >
-            {l10n.getFragment("email-footer-source-hibp", {
-              elems: {
-                "hibp-link": (
-                  <a
-                    href="https://haveibeenpwned.com"
-                    style={{ color: "#0060DF", fontWeight: 400 }}
-                  />
-                ),
-              },
-            })}
-          </mj-text>
-          <mj-text
-            font-size="14px"
-            line-height="21px"
-            font-weight="400"
-            align="center"
-          >
-            <a href={CONST_URL_TERMS} style={{ color: "#0060DF" }}>
-              {l10n.getString("terms-of-service")}
-            </a>
-            &emsp;
-            <a href={CONST_URL_PRIVACY_POLICY} style={{ color: "#0060DF" }}>
-              {l10n.getString("email-footer-meta-privacy-notice")}
-            </a>
-          </mj-text>
-        </mj-column>
-      </mj-section>
-    </mj-wrapper>
+    <mjml>
```

Comment:
I don't think you can nest `<mjml>` tags inside other `<mjml>` tags (the ones in the full email template). So I think the `<mj-style>` will have to be moved into the `<mj-head>` of every individual email :/ 

I also would instead try to hide the image in dark mode, rather than adding it in light mode - with dark mode being the edge case, and media queries (and anything that's not native MJML functionality) being unlikely to work consistently across email clients, that ensures that it's both less likely to go wrong, and when it does go wrong, that only the rare case breaks.

Reply:
Yeah 😢

Reply:
Yeah up to you if it's worth the hassle, depending on how important product considers this to be :)